### ISSUE TITLE
question about test

### ISSUE THREAD

**Issue by ygjwd12345** [W-zx-Y/DANNet#4]
Could you share more detail on how to successfully submit to the server?

**Comment by ygjwd12345** [comment]
I always get error like
Missing prediction for ground truth file GOPR0364_frame_000001_gt_labelTrainIds.png

**Comment by W-zx-Y** [comment]
> Could you share more detail on how to successfully submit to the server?

Hi,
Sorry for my late response. I can show you my submission folder.
First, I prepared three folders.
![image](https://user-images.githubusercontent.com/75738803/128374907-7c726b47-1c75-4e3b-833a-db9cbab7954b.png)
The one in the middle stores all the predictions, and their filenames are like this. I thought there might be something wrong with your filename.
![image](https://user-images.githubusercontent.com/75738803/128375327-0e1ded26-092f-4947-b5e1-51a5cd6ac719.png)

Since I did not explore the computation of UIoU, I just randomly put some figures in the confidence folder. The number and names of the figures are the same as that of the second folder. And the third folder contains exactly the same things as the second one. 

And then directly compress the three folders. Let me know if there is anything unclear.
